Releasing.md

```diff
@@ -49,6 +48,10 @@ cut a new release.
 <p>o Run the release script in release-toolkit-gutenberg-mobile: <code>./release_automation.sh</code>. This will take care of creating the gutenberg and gutenberg-mobile release PRs as well as WPAndroid and WPiOS integration PRs.</p>
 <!-- /wp:paragraph -->
 
+<!-- wp:paragraph -->
+<p>o Message any related Slack channels to inform that the gutenberg-mobile release is now cut and any new WPiOS and WPAndroid changes having related gutenberg-mobile or gutenberg parts should now be merged to <code>gutenberg/after_X.XX.X</code> branches until WPiOS and WPAndroid have cut their own releases next week.</p>
+<!-- /wp:paragraph -->
```

Comment:
@cameronvoell I'm not sure if we could automate this instead?

Reply:
I'll add a task to my queue for looking into the best way to automate this